src/components/tables/tests/TableChildBody.spec.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
```

Comment:
Only renamed?

Reply:
The props used to be the whole table props, I edited `TableChildBody` so that it is simpler to test. So all tests have changed to only get what they need as props. And I've added a few more.